src/Control/Inciter/InputDeck/InputDeck.h

```diff
@@ -75,8 +75,6 @@ class InputDeck :
                                        kw::pde_diffusivity,
                                        kw::pde_lambda,
                                        kw::pde_u0,
-                                       kw::poisson,
-                                       kw::dir_neu,
```

Comment:
If these are particle specific (and that's why they were removed), is there a nice logical way to group some of these things together?

Reply:
They are not particle-related. The history of this is that (around a year ago) I wanted to do some research in algebraic multi-grid on top of Charm++, i.e., distributed-memory, asynchronous, AMG and a Poisson equation with inhomogeneous Dirichlet and Neumann boundary conditions (for which I know the analytical solution) would have been a good test problem. I just remove it now, because this is not going to happen in the next couple of years at least.